tools/classes.js

```diff
@@ -346,6 +346,7 @@ var classes = [
 
 
   // Testing support
+  'foam.test.AllProperties',
```

Comment:
I don't think the test requires the AllProperties class, as it serializes AllSerializableProperties.